pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -48,8 +59,8 @@ func NewCgroupManager(cs *cgroupSubsystems) CgroupManager {
 // Exists checks if all subsystem cgroups already exist
 func (m *cgroupManagerImpl) Exists(name string) bool {
```

Comment:
For this it would essentially have to verify that all the values in all the cgroups are same as in the cgroupConfig. And libcontainer doesn't provide a way of doing that.  In a scenario where kubelet restarts after creating cgroups but before updating cgroup values, when it restarts we apply the pod limits on it again. So it should not be a problem.